app/_hub/kong-inc/saml/_index.md

```diff
@@ -250,7 +276,25 @@ params:
         - `Lax`: Cookies are not sent on normal cross-site subrequests, like loading images or frames into a third party site, but are sent when a user is navigating to the origin site, like when they are following a link.
         - `None`: Cookies will be sent in all contexts, including responses to both first-party and cross-origin requests. If `SameSite=None` is set, the cookie secure attribute must also be set or the cookie will be blocked.
         - `off`: Do not set the Same-Site flag.
+    - name: session_cookie_same_site
+      minimum_version: "3.2.x"
+      required: false
+      default: Lax
+      datatype: string
+      description: |
+        Controls whether a cookie is sent with cross-origin requests, providing some protection against cross-site request forgery attacks:
+        - `Strict`: Cookies will only be sent in a first-party context and not be sent along with requests initiated by third party websites.
+        - `Lax`: Cookies are not sent on normal cross-site subrequests, like loading images or frames into a third party site, but are sent when a user is navigating to the origin site, like when they are following a link.
+        - `None`: Cookies will be sent in all contexts, including responses to both first-party and cross-origin requests. If `SameSite=None` is set, the cookie secure attribute must also be set or the cookie will be blocked.
+        - `Default`: Do not explicitly specify a Same-Site attribute.
```

Comment:
Wouldn't the default be `Lax`? Or is `Default` a separate option from `Lax`?

Reply:
I know. Sounds weird, but `Default` is in fact a separate option which... is not the default :smile: